### ISSUE TITLE
missing /bin/sh after upgrading to v2.0.0.2

### ISSUE THREAD

**Issue by nbari** [ansible/ansible#14399]
On a Mac OS X 'Yosemite' I upgraded to v.2.0.0.2 and wasn't available to SSH anymore, using -vvv i got this:

`ssh -C -q -o ControlMaster=auto -o ControlPersist=60s -o KbdInteractiveAuthentication=no -o PreferredAuthentications=gssapi-with-mic,gssapi-keyex,hostbased,publickey -o PasswordAuthentication=no -o User=devops -o ConnectTimeout=10 -o ControlPath=/Users/nbari/.ansible/cp/ansible-ssh-%h-%p-%r 10.0.8.4 'LANG=en_US.UTF-8 LC_ALL=en_US.UTF-8 LC_MESSAGES=en_US.UTF-8 /usr/local/bin/python'`

comparing the output of ansible v1.9.4 I notice that there is a missing /bin/sh before the`'LANG=en_US.UTF-8....'`

for now I had to rollback to 1.9.4

**Comment by jimi-c** [comment]
Hi @nbari, we've recently addressed this issue in the devel/stable-2.0 branches, if you could please retest against one of those to verify if your issue is resolved?

**Comment by nbari** [comment]
Hi, how could I install the devel/stable-2.0 branch using pip ?

**Comment by sivel** [comment]
```
pip install -U git+https://github.com/ansible/ansible.git@devel#egg=ansible
```

Replace `devel` with `stable-2.0` if you want the 2.0 branch instead of 2.1.

**Comment by jimi-c** [comment]
As this has been resolved in devel/stable-2.0, I'm going to go ahead and close this now. If you have any further questions, please let us know by stopping by one of the two mailing lists, as appropriate:
- https://groups.google.com/forum/#!forum/ansible-project - for user questions, tips, and tricks
- https://groups.google.com/forum/#!forum/ansible-devel - for strategy, future planning, and questions about writing code

Because this project is very active, we're unlikely to see comments made on closed tickets, but the mailing list is a great way to ask questions, or post if you don't think this particular issue is resolved.

Thank you!
